lib/run/index.js

```diff
@@ -215,6 +217,22 @@ module.exports = function (options, callback) {
             // add non generic callback handling
             _.assignIn(callbacks, {
 
+                /**
+                 * Emits event for start of the run. It injects/exposes additional objects useful for
+                 * programmatic usage and reporters
+                 *
+                 * @param {?Error} err - An Error instance / null object.
+                 * @param {Object} cursor - The run cursor instance.
+                 * @returns {*}
+                 */
+                start (err, cursor) {
+                    emitter.emit('start', null, {
+                        cursor: cursor,
+                        run: run,
+                        error: err
+                    });
```

Comment:
Seems like a breaking change we can avoid. 😄 

```suggestion
                start (err, cursor) {
                    emitter.emit('start', err, {
                        cursor: cursor,
                        run: run
                    });
```

Reply:
True. Stupid me. 🙈